app/javascript/app/data/constants.js

```diff
@@ -2,11 +2,13 @@ const { GFW_API } = process.env;
 const { CW_API } = process.env;
 const { ESP_API } = process.env;
 
-export const LOGIN_URL = `${GFW_API}/auth/login?applications=climate-watch&token=true&callbackUrl=${location.origin}${CW_API}/auth/login`;
+export const loginURL = social =>
```

Comment:
Seems weird to have a function inside a file called constants, maybe we could keep the constant here and then add a getLoginUrlBySocial helper